### PR TITLE
Update links from VC-JWT to VC-JOSE-COSE

### LINKED ISSUE #1233: [VC-JWT] Reference is wrong
[VC-JWT] reference (https://www.w3.org/TR/vc-data-model-2.0/#bib-vc-jwt) redirects to a 404 page (https://w3c.github.io/vc-jwt/)

### PR DESCRIPTION
**Pull request by andresuribe87**
Fixes #1233

***
<a href="https://pr-preview.s3.amazonaws.com/andresuribe87/vc-data-model/pull/1234.html" title="Last updated on Aug 20, 2023, 7:20 PM UTC (97bdc05)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/vc-data-model/1234/6d626f8...andresuribe87:97bdc05.html" title="Last updated on Aug 20, 2023, 7:20 PM UTC (97bdc05)">Diff</a>

### CODE REVIEW

**Review by decentralgabe — APPROVED** [review]

**Review by brentzundel — APPROVED** [review]

File: index.html [hunk 1]
```diff
@@ -2571,7 +2571,7 @@ <h3>Extensibility</h3>
           </li>
           <li>
 Support multiple types of cryptographic proof formats through the use of
-JSON Web Tokens [[VC-JWT]] and Data Integrity Proofs [[VC-DATA-INTEGRITY]] and a variety of cryptographic
+JOSE or COSE [[VC-JOSE-COSE]] and Data Integrity Proofs [[VC-DATA-INTEGRITY]] and a variety of cryptographic
```
Comment (TallTed) [inline index.html:2574]:
```suggestion
JOSE or COSE [[VC-JOSE-COSE]], Data Integrity Proofs [[VC-DATA-INTEGRITY]], and a variety of cryptographic
```

**Review by TallTed — CHANGES_REQUESTED** [review]
Small tweak for clarity

**Comment by decentralgabe** [comment]
there are some other references to vc-jwt that should be updated like the text here https://github.com/w3c/vc-data-model/blob/main/index.html#L4050 and here https://github.com/w3c/vc-data-model/blob/main/index.html#L583

**Comment by TallTed** [comment]
@decentralgabe -- I think https://github.com/w3c/vc-data-model/blob/main/index.html#L583 moved to https://github.com/w3c/vc-data-model/blob/main/index.html#L591

**Comment by andresuribe87** [comment]
> there are some other references to vc-jwt that should be updated like the text here https://github.com/w3c/vc-data-model/blob/main/index.html#L4050 and here https://github.com/w3c/vc-data-model/blob/main/index.html#L583

Both changes were already captured. I've merged from main so you can see the same lines.

**Review by OR13 — APPROVED** [review]

**Review by dlongley — APPROVED** [review]

**Review by shigeya — APPROVED** [review]

**Comment by iherman** [comment]
The issue was discussed in a [meeting](https://www.w3.org/2017/vc/WG/Meetings/Minutes/2023-08-15-vcwg-special#section1-5) on 2023-08-15

- no resolutions were taken

<details><summary><i>View the transcript</i></summary>

#### 1.5. Update links from VC-JWT to VC-JOSE-COSE (pr vc-data-model#1234)

_See github pull request [vc-data-model#1234](https://github.com/w3c/vc-data-model/pull/1234)._




**Brent Zundel:** next PR: 1234 Update links from VC-JWT to VC-JOSE-COSE. request for changes from Ted. believe they've been addressed.  

**Gabe Cohen:** A note that Ted's comments were resolved, I think we're good to merge this.  

</details>

**Review by msporny — APPROVED** [review]

**Comment by msporny** [comment]
Normative, multiple reviews, changes requested and made, no objections, merging.

**Pull request merged**